docs/serving/using-subroutes.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Creating and using Subroutes"
+linkTitle: "Configuring cluster-local services"
+weight: 20
+type: "docs"
+---
+
+Subroutes are most effective when used with multiple revisions. When defining a knative service/route, the traffic section of the spec can split between the different revisions. For example:
+
+```yaml
+traffic:
+- percent: 0
+  revisionName: foo
+- percent: 40
+  revisionName: bar
+- percent: 60
+  revisionName: baz
+```
+
+This allows anyone targeting the main route to have a 0% chance of hitting revision `foo`, 40% chance of hitting revision `bar` and 60% chance of hitting revision `baz`.
+
```

Comment:
```suggestion

## Using tags to create target URLs

```

Reply:
This has been added, Github didn't detect it.